magefile.go

```diff
@@ -298,3 +312,85 @@ func InstallKO(version string) error {
 
 	return archive.DownloadToGopathBin(opts)
 }
+
+func DownloadLicenseData() error {
```

Comment:
Implement this function using the license downloader package (see the comment about keeping it below). It also does the download in parallel which makes it run faster.

Reply:
Actually the code has been copied from that package, however I was getting mysterious errors with parallelism. Essentially the json of some downloaded files was getting malformed. Maybe there's some race. 

I think for the first draft, non-parallel code is fine, it's not a performance bottleneck anymore since we download only once, that too in CI